apps/prairielearn/src/middlewares/studentAssessmentAccess.ts

```diff
@@ -28,11 +29,12 @@ router.all('/', function (req, res, next) {
       !_.get(res.locals, 'authz_result.active', true))
   ) {
     // This assessment instance is closed and can no longer be viewed
-    if (!_.get(res.locals, 'authz_result.show_closed_assessment_score', true)) {
-      closedAssessmentNotActiveHiddenGrade(res);
-    } else {
-      closedAssessmentNotActive(res);
-    }
+    res.status(403).send(
+      StudentAssessmentAccess({
+        resLocals: res.locals,
+        showClosedScore: _.get(res.locals, 'authz_result.show_closed_assessment_score', true),
```

Comment:
Nit: optional chaining and nullish coalescing?

Reply:
831a487fe